src/content/developers/docs/apis/json-rpc/index.md

```diff
@@ -411,7 +456,7 @@ curl -X POST --data '{"jsonrpc":"2.0","method":"eth_mining","params":[],"id":71}
 
 ### eth_hashrate {#eth_hashrate}
 
-Returns the number of hashes per second that the node is mining with.
+Returns the number of hashes per second that the node is mining with. This can only return `true` for proof-of-work networks and may not be available in some clients since The Merge to proof of stake.
```

Comment:
Same. Also proof-of-stake, not proof of stake.

Reply:
```suggestion
Returns the number of hashes per second that the node is mining with. This can only return `true` for proof-of-work networks and may not be available in some clients since [The Merge](/roadmap/merge/).
```